src/common/modules/data/Tips.js

```diff
@@ -118,7 +131,8 @@ const tipArray = [
 
             // find command
             const allCommands = await browser.commands.getAll();
-            const popupOpenCommand = allCommands.find((command) => command.name === "_execute_browser_action");
+            const acommand = IS_THUNDERBIRD ? "_execute_compose_action" : "_execute_browser_action";
+            const popupOpenCommand = allCommands.find((command) => command.name === acommand);
```

Comment:
```suggestion
            const commandToFind = IS_THUNDERBIRD ? "_execute_compose_action" : "_execute_browser_action";
            const popupOpenCommand = allCommands.find((command) => command.name === commandToFind);
```

Still don't like that `a` prefix that is not self-explanatory, better find a good variable name.